tests/input/cpp/indent_ctor_init.cpp

```diff
@@ -1,7 +1,12 @@
-struct MyClass {
+struct MyClass : public Foo,
+                 private Bar {
   MyClass(int a, int b, int c)
 	 : m_a(a), m_b(b),
    m_c(c) {}
 private :
   int m_a, m_b, m_c;
 };
```

Comment:
It is not pretty to declare more than one variable in a line.
Please split in many lines.